flashcards/backend/decks/models.py

```diff
@@ -31,15 +31,20 @@ class Meta:
 class Category(models.Model):
     """Model for representing Category for deck
 
+    :param image: Category image
+    :type image: file
     :param name: Category name
     :type name: str
     :param description: Description for category
     :type description: str
 
     """
 
-    name: models.CharField = models.CharField(max_length=24, unique=True)
-    description: models.TextField = models.TextField()
+    image = models.FileField(
```

Comment:
Please confirm if we should have image also in the Category model as it's not specified on ER diagram
![Screenshot from 2023-04-25 22-43-43](https://user-images.githubusercontent.com/94061802/234592717-bd545c1b-66ea-4333-bce7-6201ec76effd.png)

Reply:
if DB schema is modified during work then ER diagram should be updated